ods_ci/tests/Resources/Common.robot

```diff
@@ -305,3 +305,38 @@ Extract URLs From Text
     [Arguments]    ${text}
     ${urls}=     Get Regexp Matches   ${text}   (?:(?:(?:ftp|http)[s]*:\/\/|www\.)[^\.]+\.[^ \n]+)
     RETURN    ${urls}
+
+Run And Watch Command
+  [Documentation]    Run any shell command (including args) with optional:
+  ...    Timeout in minutes: 10 by default.
+  ...    Expected text: Verify the text exists in command output.
+  [Arguments]    ${command}    ${timeout_min}=10    ${expected_text}=${NONE}
+  Log    Watching command output: ${command}   console=True
+  @{args} =    Split String    ${command}
+  ${process_log} =    Set Variable    ${OUTPUT DIR}/${args}[0].log
+  Create File    ${process_log}
+  Create File    ${process_log}.old
+  ${process} =    Start Process    ${command}    shell=True    stdout=${process_log}    stderr=STDOUT    # robocop: disable
+  Log    Shell process started in the background   console=True
+  ${timeout_result} =    Wait Until Keyword Succeeds    ${timeout_min} min    10 s
+  ...    Read Command Log    ${process}    ${process_log}
```

Comment:
what is it doing? does it wait until the first log line is written?

Reply:
oh it is kind of "wait until the process finished to write logs". Is that correct? 
If that so, I think the Wait Until Kw Succeeds is not necessary here, unless there is the chance it fails when the file is empty, hmm

Reply:
I think the set of naming here can be misleading, because I wouldn't expect a kw named `Read Command Log` to also to assess if the process is ended

Reply:
ack, I renamed "Read Command Log" to "Check Process Output and Status"